addons/stock/models/stock_picking.py

```diff
@@ -150,6 +150,10 @@ class PickingType(models.Model):
         compute_sudo=True, string='Show Operation in Overview'
     )
     kanban_dashboard_graph = fields.Text(compute='_compute_kanban_dashboard_graph')
+    shipping_policy = fields.Selection([
```

Comment:
nitpick: maybe we could also use move_type here as the name of the field, so it mirrors the existing fields on stock.picking and procurement.group?

Reply:
I've initially wanted to change the other fields to `shipping_policy`, as this seems to be how we usually call this :P But, seeing how updating the existing fields is a headache, I guess I can use `move_type` here, for consistency ;)